Makefile

```
@@ -7,57 +7,62 @@ GOBUILD=CGO_ENABLED=0 go build -trimpath -ldflags ' \
 		-X "main.BuildTime=$(BUILDTIME)" \
 		-w -s -buildid='
 
-all: linux-amd64 linux-arm64 freebsd-amd64 freebsd-arm64 darwin-amd64 darwin-arm64 win64 win32
+all: linux-amd64 linux-arm64 freebsd-amd64 freebsd-arm64 darwin-amd64 darwin-arm64 windows-amd64 windows-386
```

checkmake (0.3.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)